### ISSUE TITLE
Figure out how to set the initial position of coin that animates from prep area to measurement area

### ISSUE THREAD

**Issue by jbphet** [phetsims/quantum-measurement#11]
In the "Coins" screen, when the user switches from preparing the experiment to measuring outcomes, coins are supposed to animate from the prep area to the measurement area.  I'm having trouble reconciling the various coordinate frames necessary to set the initial position for this animation, so I'll use some hardcoded values for now and see if I can get help on how to do this properly.

**Comment by jbphet** [comment]
Deferring until this sim becomes a priority again, which should be in a month or so.

**Comment by jbphet** [comment]
I have run some experiments to try to figure out how to make this work, and I learned this: If I get the global bounds of a scenery Node - in this case the `orientationIndicatorCoinNode` in the `InitialCoinStateSelectorNode` class - and use those bounds to set the position of a different Node in the global coordinate frame, they don't end up in the same place.  This surprises me, so perhaps I have a misunderstanding somewhere along the way.  I've reached out to @jonathanolson for some consultation on this, and I'll pick it up again when he has some time to help me out.

**Comment by jbphet** [comment]
This has been addressed.  The tricky bits were:
- The global coordinate frame changes with the window size.  I didn't realize this - I thought the global coordinate frame was essentially based on the dev bounds.  So, I changed the code to translate between global and local coordinate frames right away.
- There was a bit of a race condition between figuring out the origin for the coin movement animations and the changes to the size of the preparation area.  I added a variable to track the origin location and update it at the end of the animations, and this seems to work well.

Closing.
